package.json

```diff
@@ -49,53 +45,54 @@
     "lodash.merge": "^3.3.2",
     "lodash.restparam": "^3.6.1",
     "mime-types": "^2.1.8",
-    "mocha": "^2.3.4",
-    "mocha-phantomjs": "^4.0.2",
+    "mocha": "^5.2.0",
+    "mocha-chrome": "^1.1.0",
```

Comment:
Have you considered adding https://github.com/GoogleChrome/puppeteer to not be dependant on the installed (or not installed) Chrome?

Reply:
I'm not familiar with Puppeteer but it sounds like it's overkill for what we need at this point. `mocha-chrome` runs headless Chrome for the tests. I updated the Travis config to include Chrome and that seems to be sufficient.

Reply:
Yeah Puppeteer needs to download ~80 MB with every `npm install` :-P